services/podcast_results_service.rb

```diff
@@ -4,23 +4,21 @@
 require 'dotenv'
 Dotenv.load
 
-class PodcastResultsService 
+class PodcastResultsService
 
 
-  def podcast(podcast_id) 
+  def podcast(podcast_id)
     get_json("/podcasts/#{podcast_id}")
   end
-  
-  def conn 
-    Faraday.new("https://listen-api.listennotes.com/api/v2") do |faraday|
-      faraday.headers['X-ListenAPI-Key'] = ENV["LISTEN_API"]
 
+  def conn(url)
```

Comment:
Take notice of order of operations being different than previous examples.  This allowed us to use the API dynamically.